src/nnet3/nnet-diagnostics.h

```diff
@@ -108,7 +118,7 @@ class NnetComputeProb {
   void Compute(const NnetExample &eg);
 
   // Prints out the final stats, and return true if there was a nonzero count.
-  bool PrintTotalStats() const;
```

Comment:
This is supposed to be used in nnet3-show-progress, but since that has a bug and cannot be used in multilingual setup, this change is not useful anyway.
I have not changed any semantics. I just add tot_weight that could be used in nnet3-show-progress.